docs/core/porting/upgrade-assistant-winforms-framework.md

```diff
@@ -6,6 +6,42 @@ ms.date: 11/08/2021
 ---
 # Upgrade a Windows Forms App to .NET 6 with the .NET Upgrade Assistant
 
+
+<!--
```

Comment:
It looks like the changes here are commented out. Do they show up in the rendered Markdown?

Reply:
This file is in progress